compute/gpu/how-to/create-manage-gpu-instance.mdx

```diff
@@ -6,11 +6,11 @@ content:
   h1: How to create and manage a GPU Instance
   paragraph: This page explains how to create and manage a GPU Instance
 dates:
-  validation: 2021-12-14
-  posted: 2021-12-14
+  validation: 2022-03-25
+  posted: 2022-03-25
 ---
 
-Scaleway GPU Instances are virtual compute Instances equipped with dedicated high-end Nvidia graphical processing unit (GPUs). They are ideal for data processing, artificial intelligence, rendering and video encoding. After you have created your GPU Instance, you can connect to it via SSH and use one of our ready-made [Docker images](/compute/gpu/how-to/use-gpu-with-docker/) to access a [preinstalled environment](/compute/gpu/how-to/use-preinstalled-env/) with all your favorite AI libraries and tools pre-installed. In addition to this, GPU Instances have all the features of our regular Instances, including [flexible IPs](/compute/instances/how-to/use-flexips/), [security groups](/compute/instances/how-to/use-security-groups/), [Private Networks](/compute/instances/how-to/use-private-networks/), [backups](/compute/instances/how-to/create-a-backup/) and more. When you are done using your GPU Instance, you can easily delete it from the Scaleway Console.
+Scaleway GPU Instances are virtual compute instances equipped with dedicated high-end Nvidia graphical processing unit (GPUs). They are ideal for data processing, artificial intelligence, rendering and video encoding. After you have created your GPU Instance, you can [connect to it via SSH](/compute/instances/how-to/connect-to-instance) and run one of our ready-made [Docker images](/compute/gpu/how-to/use-gpu-with-docker) to access a [preinstalled environment](/compute/gpu/how-to/use-preisntalled-env) environment with all your [favorite AI libraries and tools](/compute/gpu/reference-content/docker-images/) pre-installed. In addition to this, GPU Instances have all the features of our regular Instances, including [flexible IPs](/compute/instances/how-to/use-flexips/), [Security Groups](/compute/instances/how-to/use-security-groups/), [Private Networks](/compute/instances/how-to/use-private-networks/), [backups](/compute/instances/how-to/create-a-backup/) and more. When you are done using your GPU Instance, you can easily [delete it](/compute/instances/how-to/delete-instance/) from the Scaleway console.
```

Comment:
Shall we mention the deletion of the Instance in the intro?

Reply:
I think its worth leaving in for AI devs who aren't used to working in the cloud, might find it reassuring that you can delete at any time?